File: codingTest/programmers/javascript/행렬의 덧셈.js
var solution = (arr1, arr2) => {
  var result = [];
  for (var i in arr1) {
    var row = [];
    for (var j in arr1[i]) row.push(arr1[i][j] + arr2[i][j]);
    result.push(row);
  }
  return result;
};

File: codingTest/programmers/python/행렬의 덧셈.py
def solution(arr1, arr2):
    result = []
    for i in range(len(arr1)):
        row = []
        for j in range(len(arr1[i])):
            row.append(arr1[i][j] + arr2[i][j])
        result.append(row)
    return result